docs/concepts/whatisvpp/performance.rst

```diff
@@ -0,0 +1,35 @@
+.. _performance:
+
+.. toctree::
+
+Performance
+***********
+
+Performance Expectations
+-------------------------
+
+One of the benefits of this implementation of FD.io VPP is its high
+performance on relatively low-power computing. This high level of
+performance is based on the following highlights:
+
+* High-performance user-space network stack for commodity hardware
+* The same code for host, VMs, Linux containers
+* Integrated vhost-user virtio backend for high speed VM-to-VM connectivity
+* L2 and L3 functionality, multiple encapsulations
+* Leverages best-of-breed open source driver technology: DPDK
+* Extensible by use of plugins
+* Control-plane / orchestration-plane via standards-based APIs
+
+Performance Metrics
+-------------------
+
+The FD.io VPP platform has been shown to provide the following approximate
```

Comment:
Ok so we need to write the "Performance Metrics" section a bit. 

FD.io VPP platform is a high performance packet processing software, typically achieving: